### ISSUE TITLE
Support for Win 11 22H2 build 22621.382

### ISSUE THREAD

**Issue by ffrags** [ChrisRfr/Win10XPE#35]
Hello again ChrisR,

Hope all is good and that you are well. It's been a while since my last post. I am hoping that you have some little time to refresh the Jan 7, 2022 files as they do not seem to  work when using yesterday's release of Windows 11 build 22621.382 from TechBench.

Through some forum research, I found that if I go to the "Additions" section and within the "Add Windows 10 Files and Folders" add the following 2 lines

\Windows\System32\Windows.UI.Core.TextInput.dll
\Windows\System32\windowsudk.shellcommon.dll

then that allows my build to boot to a stable desktop and have all applications operate normally.

Without those two lines, the build will not boot to desktop or it will get a quickly flashing message saying "Hold ESC key to prevent Startls from loading".

Also, the log file for a good build shows a new error message under the "Warnings" heading that states:
Error: Wimlib Extract mrxsmb10.sys from Install.wim Return: 49 - File Not Found
Not sure if that is all that important.

Lastly, with the 2 lines added and booting to the stable Win10XPE desktop, there is a message box appearing in the lower right which says "Language preferences". The box will remain there until I click on it and then it disappears and another box pops up in upper left which says "You'll need a new app to open this ms-settings link".

I would be nice to have a complete refresh of the project available for download that would have built in support for 22H2 not have the "Language preferences" box appear.

Thank you for your continuing support and miss not seeing you more often in the old forums.

**Comment by starchivore** [comment]
[https://www.tenforums.com/software-apps/182581-win10xpe-build-your-own-rescue-media-2-a-99.html#post2467420](https://www.tenforums.com/software-apps/182581-win10xpe-build-your-own-rescue-media-2-a-99.html#post2467420)
>The sound does not seem to work, it is not sent to the audio devices! probably linked to MMDevices API, AudioEndpointBuilder.dll...

FYI - @slorelee mentioned that `Administrator` account should be required for audio devices to work properly under Windows 11 build 22621.1 or above:

[http://bbs.c3.wuyou.net/forum.php?mod=redirect&goto=findpost&ptid=411399&pid=4640550](http://bbs.c3.wuyou.net/forum.php?mod=redirect&goto=findpost&ptid=411399&pid=4640550)
>22621的声音目前需要登录Admin账户才可以。

In other words, maybe we won't get anywhere with `NT AUTHORITY\SYSTEM` then?

**Comment by ChrisRfr** [comment]
Slore is an expert. If he writes this, there is a good chance of not being able to get the sound with the System account.
yet, strange but the narrator works!

**Comment by angelics** [comment]
nt sure if its hijacking but this issue only exist in 22621.382, powershell is nt working even with .netfx4 included.
verified with dotnet.exe that its installed.
`Version v4.0.30319 of the .NET Framework is not installed and it is required to run version 3 of Windows Poweshell`

**Comment by starchivore** [comment]
Hooray, `Noel-Blanc` finally cracked the code while the culprit turned out to be the file `AudioSrvPolicyManager.dll`:

[https://theoven.org/viewtopic.php?p=1215#p1215](https://theoven.org/viewtopic.php?p=1215#p1215)

[http://bbs.c3.wuyou.net/forum.php?mod=viewthread&tid=434900](http://bbs.c3.wuyou.net/forum.php?mod=viewthread&tid=434900)
